### PR TITLE
Corrected config documentation

### PR DESCRIPTION
**Pull request by matsduf**
* Corrected POD in code
* Corrected links in documentation

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ TBD
 
 --------
 
-[Profile JSON files]: https://metacpan.org/pod/Zonemaster::Engine::Config#PROFILE-DATA
+[Profile JSON files]: https://metacpan.org/pod/Zonemaster::Backend::Config
```
Comment (mattias-p) [inline docs/Configuration.md]: The link should be https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md

File: docs/Configuration.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ TBD
...
 [Profile names]: API.md#profile-name
 [Profiles]: Architecture.md#profile
-[Zonemaster Engine default profile]: https://metacpan.org/pod/Zonemaster::Engine::Config#DESCRIPTION
+[Zonemaster Engine default profile]: https://metacpan.org/pod/Zonemaster::Backend::Config
```
Comment (mattias-p) [inline docs/Configuration.md]: The link should be https://metacpan.org/pod/Zonemaster::Engine::Profile#PROFILE-PROPERTIES

File: docs/Configuration.md (hunk 1 above)
Comment (sandoche2k) [reply docs/Configuration.md]: The link to default profile in the above link is also missing : https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md

**Comment by matsduf** [comment]
@mattias-p and @sandoche2k, please review again after update.

File: docs/Configuration.md [hunk 3]
```diff
@@ -40,22 +40,22 @@ the [Zonemaster Engine default profile].
...
 TBD
 
 --------
 
-[Profile JSON files]: https://metacpan.org/pod/Zonemaster::Engine::Config#PROFILE-DATA
+[Profile JSON files]: https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md
 [Profile names]: API.md#profile-name
 [Profiles]: Architecture.md#profile
-[Zonemaster Engine default profile]: https://metacpan.org/pod/Zonemaster::Engine::Config#DESCRIPTION
+[Zonemaster Engine default profile]: https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md
```
Comment (sandoche2k) [inline docs/Configuration.md]: The above link is the same as the earlier one. The default profile link in the page "https://github.com/zonemaster/zonemaster-engine/blob/master/share/default.profile" returns a 404 error

Comment (matsduf) [reply docs/Configuration.md]: Please see https://github.com/zonemaster/zonemaster-engine/pull/687

**Comment by matsduf** [comment]
@mattias-p and @sandoche2k, please review again.

**Comment by matsduf** [comment]
@sandoche2k, can you review again?

**Comment by sandoche2k** [comment]
@matsduf merge?

**Pull request merged**